packages/manager/src/views/SPAPropertyDetailPage/VirtualPath/components/DeleteVirtualPath.tsx

```diff
@@ -0,0 +1,273 @@
+      onClose();
+    } catch (error) {
+      if (error instanceof AxiosError && error.response && error.response.status === 403) {
+        toast.error("You don't have access to perform this action");
+        onClose();
+      } else {
+        toast.error('Failed to delete virtual path');
+      }
+    }
+  };
+
+  return (
+    <Form onSubmit={handleSubmit(onSubmit)}>
+      <p>Please confirm if you want to delete the virtual path with the following details.</p>
+      <Split hasGutter>
+        <SplitItem isFilled style={{ width: '100%' }}>
+          <Controller
+            control={control}
+            name="propertyIdentifier"
+            defaultValue={propertyIdentifier}
+            render={({ field, fieldState: { error } }) => (
+              <FormGroup
+                label="Web property"
+                isRequired
+                fieldId="propertyIdentifier"
+                validated={error ? 'error' : 'default'}
+                helperTextInvalid={error?.message}
+              >
+                <TextInput
+                  isRequired
+                  placeholder="Add web-property name"
+                  type="text"
+                  id="propertyIdentifier"
+                  value={field.value}
+                  onChange={field.onChange}
+                  onBlur={field.onBlur}
+                  isDisabled
+                />
+              </FormGroup>
+            )}
+          />
+        </SplitItem>
+        <SplitItem isFilled style={{ width: '100%' }}>
+          <Controller
+            control={control}
+            name="identifier"
+            defaultValue={identifier}
+            render={({ field, fieldState: { error } }) => (
+              <FormGroup
+                label="Application Name"
+                isRequired
+                fieldId="identifier"
+                validated={error ? 'error' : 'default'}
+                helperTextInvalid={error?.message}
+              >
+                <TextInput
+                  isRequired
+                  placeholder="Add application name"
+                  type="text"
+                  id="identifier"
+                  value={field.value}
+                  onChange={field.onChange}
+                  onBlur={field.onBlur}
+                  isDisabled
+                />
+              </FormGroup>
+            )}
+          />
+        </SplitItem>
+      </Split>
+      <Split hasGutter>
+        <SplitItem isFilled style={{ width: '100%' }}>
+          <Controller
+            control={control}
+            name="basePath"
+            defaultValue={data?.basePath}
+            render={({ field, fieldState: { error } }) => (
+              <FormGroup
+                label={
+                  <>
+                    Base Path
+                    <Tooltip content="Symlink virtualPath path should be /{spa-path}/{virtualPath-path}">
+                      <span>
+                        &nbsp; <InfoCircleIcon style={{ color: '#6A6E73' }} />
```

Comment:
Remove this tooltip, as it's not related to the virtual path, or modify accordingly

Reply:
done